frameit/lib/frameit/editor.rb

```diff
@@ -106,19 +106,23 @@ def should_add_title?
     def complex_framing
       background = generate_background
 
+      self.top_space_above_device = vertical_frame_padding
+
+      if fetch_config['title']
+        background = put_title_into_background(background)
+      end
+
       if self.frame # we have no frame on le mac
         resize_frame!
         put_into_frame
 
         # Decrease the size of the framed screenshot to fit into the defined padding + background
         frame_width = background.width - horizontal_frame_padding * 2
+        frame_height = background.height - top_space_above_device - vertical_frame_padding
```

Comment:
What is the motivation for this height computation change (and the one a few lines below)?